### PR TITLE
Unhide Stack Transform

### PR DESCRIPTION
**Pull request by invokesus**

### CODE REVIEW

File: examples/specs/rect_mosaic_labelled_with_offset.vl.json [hunk 1]
```diff
@@ -236,7 +236,7 @@
   },
   "config": {
     "view": {
-      "stroke": null
+      "stroke": ""
```
Comment (kanitw) [inline examples/specs/rect_mosaic_labelled_with_offset.vl.json]: Why empty string here?  I think

Comment (kanitw) [reply examples/specs/rect_mosaic_labelled_with_offset.vl.json]: Same for all other examples

Comment (invokesus) [reply examples/specs/rect_mosaic_labelled_with_offset.vl.json]: Tests were failing if `stroke` was not a string.

**Comment by kanitw** [comment]
LGTM - I'm closing this so we can merge it together with the docs PR though.

**Pull request closed without merging**